### PR TITLE
[zh-cn] sync task files of task-3

### PR DESCRIPTION
**Pull request by kinzhi**
related: [#39983 ](https://github.com/kubernetes/website/issues/39983)

### CODE REVIEW

**Comment by windsonsea** [comment]
Please only include one language (such as en or zh-cn) in a PR.

File: content/zh-cn/docs/tasks/configure-pod-container/quality-service-pod.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 配置 Pod 的服务质量
 content_type: task
-weight: 30
+weight: 60
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/quality-service-pod.md:4]: 这个 frontmatter 对应的 en 需要复制过来，然后注释掉

**Review by windsonsea — COMMENTED** [review]
Thanks.
/unhold
/remove-language en

File: content/zh-cn/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 2]
```diff
@@ -94,7 +95,7 @@ Installing the above webhooks and associated objects require the steps below:
 1. 创建引用该 Deployment 的 Validating Webhook 和 Mutating Webhook 配置
 
 <!--
-A [script](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh) can be used to deploy and configure the GMSA webhooks and associated objects mentioned above. The script can be run with a `-dry-run=server` option to allow you to review the changes that would be made to your cluster.
+A [script](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh) can be used to deploy and configure the GMSA webhooks and associated objects mentioned above. The script can be run with a ```--dry-run=server``` option to allow you to review the changes that would be made to your cluster.
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-gmsa.md:98]:
```suggestion
A [script](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh) can be used to deploy and configure the GMSA webhooks and associated objects mentioned above. The script can be run with a ```--dry-run=server``` option to allow you to review the changes that would be made to your cluster.
```
这个修改有点奇怪哇

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 3]
```diff
@@ -718,14 +724,14 @@ registered or accessible.
...
-When enabled, the Kubernetes API server provides an OpenID Provider
+When enabled, the Kubernetes API server publishes an OpenID Provider
 Configuration document via HTTP. The configuration document is published at
 `/.well-known/openid-configuration`.
 The OpenID Provider Configuration is sometimes referred to as the _discovery document_.
 The Kubernetes API server publishes the related
 JSON Web Key Set (JWKS), also via HTTP, at `/openid/v1/jwks`.
 -->
-当此特性被启用时，Kubernetes API 服务器会通过 HTTP 提供一个 OpenID 提供者配置文档。
+当此特性被启用时，Kubernetes API 服务器会通过 HTTP 发表一个 OpenID 提供者配置文档。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:734]:
```suggestion
当此特性被启用时，Kubernetes API 服务器会通过 HTTP 发布一个 OpenID 提供者配置文档。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-gmsa.md (hunk 2 above)
Comment (kinzhi) [reply content/zh-cn/docs/tasks/configure-pod-container/configure-gmsa.md:98]: 与英文保持一致，中文翻译中继续保持单反引号

**Pull request merged**